linode/manager
# change: [M3-8593] - Copy updates in Create with CLI modal

## Description 📝
Highlight the Pull Request's context and intentions.

## Changes  🔄
List any change relevant to the reviewer.
- The "SDK's" tab is corrected to read "SDKs". 
- The "Getting started" header is corrected to read "Getting Started".

## Target release date 🗓️
9/30



### Verification steps
(How to verify changes)
- Navigate to API CLI Modal in linode create flow
- Verify SDK tab and resource headings. 

## As an Author I have considered 🤔

*Check all that apply*
